source/_components/sensor.trafikverket_weatherstation.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "Trafikverket WeatherStation"
+description: "Instructions how to integrate Trafikverket WeatherStation within Home Assistant."
+date: 2018-02-01 12:06
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: trafikverket.png
+ha_category: Sensor
+ha_release: 0.63.0
+---
+
+Showing weather information for air and road temperature provided by Trafikverket in Sweden. 
+
+Usage: Get weather data if you live near a weather station, just want to have the current temperature, needing the current temperature to know when your car heating system should start or when to leave for work due to queues (because the road is frozen).
+
+##### {% linkable_title Configuration %}
+To enable this sensor, add the following lines to your `configuration.yaml`.
+
+```yaml
+sensor:
+  - platform: trafikverket_weatherstations
+    name: Trafikverket Road WeatherStation Kungälv
+    api_key: eXXcbXXXacXXXXc39XX3aXXX4aXX46XX
+    station: Kungälv
+    type: road
+```
+
+Configuration variables:
+
+- **name** (*Required*): Unique name of the device in the frontend.
+- **api_key** (*Required*): API key from Trafikverket
+- **station** (*Required*): Name of the weather station
+- **type** (*Required*): Defines which temperature you want (`air` or `road`)
+- **scan_interval** (*Optional*): How frequently to query for new data (in seconds). Defaults to 300 seconds (5 minutes)
+
+##### {% linkable_title Get API key: %}
+[https://api.trafikinfo.trafikverket.se/](https://api.trafikinfo.trafikverket.se/)
```

Comment:
Use full sentences when writing documentation.

Reply:
What do you mean? Please explain more in detail.

Reply:
Title could be "Obtaining API key" and description something like "Please visit <site> and register to obtain the API key" (maybe even explain the procedure & the limitations, if any, a bit?).

Reply:
Why want to have the description (whole sentence) as a link? Will check the procedure, if I need to mention anything. Don't think so but I will check.

Reply:
Whole sentence for all links (not http.......)? Just checking, I will change the map link as well in that case.

Reply:
It does not have to be as a link, but documentation is usually written as prose instead of just having links and listings.

Reply:
Updated, please review.